sites/partners/src/applications/ApplicationsColDefs.ts

```diff
@@ -581,5 +581,30 @@ export function getColDefs(maxHouseholdSize: number, countyCode: string) {
     )
   }
 
-  return [...defs, ...householdCols]
+  const duplicateCols = [
+    {
+      headerName: t("applications.duplicates.flaggedAsDuplicate"),
+      field: "reviewStatus",
+      sortable: false,
+      filter: false,
+      width: 125,
+      minWidth: 100,
+      valueFormatter: ({ value }) => {
+        return formatYesNoLabel(value && value === "duplicate")
```

Comment:
Only thing I noticed on this PR was that I believe this should return yes for either "pending" or "duplicate" rather than just "duplicate". The way I'm understanding the difference between these columns is that an application may be flagged as a duplicate automatically but not yet marked as a duplicate by the user which results in the pending state. Is this correct @kathyccheng?

Reply:
`flagged as duplicate` means that the application is marked as a possible duplicate
`marked as duplicate`  (when the review status === duplicate) is when the application was confirmed as a duplicate

Reply:
Has this been resolved?

Reply:
I don't believe so @emilyjablonski.  If I am understanding our conversations correctly @YazeedLoonat, the issue is that flagged as duplicated will become false if a flagged application is then marked as valid when using the review_status. I believe there are three cases where flagged as duplicate should be true: 1) Application is part of a duplicate group in pending review (would be covered by my comment above) 2) Application was part of a duplicate group in pending review but then marked as duplicate covered by PR currently) 3) Application was part of a duplicate group in pending review but then marked as valid which cannot currently be represented by the review_status. Let me know if this makes sense @KrissDrawing and what your thoughts on next steps would be @YazeedLoonat.